### PR TITLE
[Docs] Control heading levels in right nav

### PR DESCRIPTION
**Pull request by samiahmedsiddiqui**
This PR allows to control levels display in right nav for page. Following params are introduced and can be added in page params.

```yaml
rightNav:
  hideH3: true # To hide H3 and corresponding headings
  hideH4: true # To hide H4 and corresponding headings
```

### CODE REVIEW

**Review by ddhodge — APPROVED** [review]
LGTM

File: docs/layouts/partials/toc.html [hunk 1]
```diff
@@ -1,7 +1,16 @@
+{{ $tocClasses := "" }}
+{{ if $.Param "rightNav.hideH3" }}
+  {{ $tocClasses = " hide-h3" }}
+{{ end }}
+
+{{ if $.Param "rightNav.hideH4" }}
+  {{ $tocClasses = printf "%s hide-h4" $tocClasses }}
```
Comment (polarweasel) [inline docs/layouts/partials/toc.html:7]: I think I'd combine this logic into an if/elseif such that hideH4 automatically also hides H3. The flow would need to be this:

if [hideH4], set tocClasses to "hide-h3 hide-h4"
else if [hideH3], set tocClasses to "hide-h3"

**Review by ddhodge — APPROVED** [review]
LGTM

**Review by polarweasel — APPROVED** [review]
Works as advertised. Yay!

**Pull request merged**